gobblin-service/src/test/java/org/apache/gobblin/service/GobblinServiceManagerTest.java

```diff
@@ -359,40 +364,57 @@ public void testRunQuotaExceeds() throws Exception {
     try {
       this.flowConfigClient.createFlowConfig(flowConfig2);
     } catch (RestLiResponseException e) {
+      /* Note: this exception may NOT occur if the first job above is processed immediately so TEST_FLOW_ID2 will remain
```

Comment:
When is quota released? After job completion? If yes, when does this test job "complete"?